docs/getting-started/orchestration_quickstart.md

```diff
@@ -0,0 +1,162 @@
+```bash
+prefect cloud login
+```
+OR to use our open source offering, in a new terminal run to host the Prefect engine locally.
+```bash
+prefect server start
+```
+
+### Step 3: Author a Flow
+**The fastest way to get started with Prefect is to clone a test repo in GitHub or equivalent and simply add an `@flow` decorator to any python function**.
+
+!!! Tip "Quick Tips"
+    - At a minimum you need to define at least one flow function.
+    - Your flows can be segmented by introducing task (`@task`) functions, which can be invoked from within these flows.
+    - A Task represents a discrete unit of Python logic whereas Flows are more akin to parent functions accommodating a broad range of workflow logic.
+    - Flows can be called inside of other flows (we call these subflows) but a task **cannot** be run inside of another task or from outside the context of a flow.
+
+Here is an example flow that calls 2 tasks:
+
+```python
+# my_flow.py
+import httpx
+from prefect import flow, task
+
+@task(retries=3)
+def get_contributors(url):
+    response = httpx.get(url)
+    if response.status_code == 200:
+        contributors = response.json()
+        return len(contributors)
+    else:
+        raise Exception('Failed to fetch contributors.')
+
+@task(retries=4)
+def calculate_average_commits(contributors):
+    commits_url = f'https://api.github.com/repos/PrefectHQ/prefect/stats/contributors'
+    response = httpx.get(commits_url)
+    if response.status_code == 200:
+        commit_data = response.json()
+        total_commits = sum(c['total'] for c in commit_data)
+        average_commits = total_commits / contributors
+        return average_commits
+    else:
+        raise Exception('Failed to fetch commit information.')
+
+@flow(name="Repo Info", log_prints=True)
+def my_flow_function():
+    url = 'https://api.github.com/repos/PrefectHQ/prefect'
+    api_response = httpx.get(url)
+    if api_response.status_code == 200:
+        repo_info = api_response.json()
+        stars = repo_info['stargazers_count']
+        forks = repo_info['forks_count']
+        contributors_url = repo_info['contributors_url']
+        contributors = get_contributors(contributors_url) # Task Call Here
+        average_commits = calculate_average_commits(contributors) # Task Call Here
+        print(f"PrefectHQ/prefect repository statistics 🤓:")
+        print(f"Stars 🌠 : {stars}")
+        print(f"Forks 🍴 : {forks}")
+        print(f"Average commits per contributor 💌 : {average_commits:.2f}")
+    else:
+        raise Exception('Failed to fetch repository information.')
+
+if __name__ == '__main__':
+    my_flow_function()
+```
+
+### Step 4: Run your Flow Locally
+Prefect Flows don't just look pythonic, they run like python functions too! 
+
+Call any function that you've decorated with a `@flow` decorator to see a local instance of a FlowRun.
```

Comment:
This is the first that I've seen "FlowRun", elsewhere I believe its "flow run", right?